homeassistant/components/unifiprotect/translations/en.json

```diff
@@ -51,9 +51,16 @@
         }
     },
     "issues": {
-        "deprecate_smart_sensor": {
-            "description": "The unified \"Detected Object\" sensor for smart detections is now deprecated. It has been replaced with individual smart detection binary sensors for each smart detection type. Please update any templates or automations accordingly.",
-            "title": "Smart Detection Sensor Deprecated"
+        "deprecated_service_set_doorbell_message": {
+            "fix_flow": {
+                "step": {
+                    "confirm": {
+                        "description": "The `unifiprotect.set_doorbell_message` service is deprecated in favor of the new Doorbell Text entity added to each Doorbell device. It will be removed in v2023.3.0. Please update to use the [`text.set_value` service](https://www.home-assistant.io/integrations/text#service-textset_value).",
```

Comment:
We should put the url in `translation_placeholders` so the translators don't have to handle it and we can change it without having to retranslate (probably unlikely in this case though)